docs/4.2/ibm_cloud_guide.md

```diff
@@ -0,0 +1,179 @@
+# Running Teleport on IBM Cloud
+
+We've created this guide to give customers a high level overview of how to use Teleport
+on the [IBM Cloud](https://www.ibm.com/cloud). This guide provides a high level 
+introduction leading to a deep dive into how to setup and run Teleport in production.
+
+We have split this guide into:
+
+- [Teleport on IBM FAQ](#teleport-on-ibm-cloud-faq)
+- [IBM Teleport Introduction](#ibm-teleport-introduction)
+
+## Teleport on IBM Cloud FAQ
+
+#### Why would you want to use Teleport with IBM Cloud?
+Teleport provides privileged access management for cloud-native infrastructure that 
+doesn’t get in the way. Infosec and systems engineers can secure access to their 
+infrastructure, meet compliance requirements, reduce operational overhead, and have 
+complete visibility into access and behavior.
+
+By using Teleport with IBM you can easily unify all access for both IBM Cloud and 
+softlayer infrastrcure. 
```

Comment:
I think `Softlayer` is a proper noun here as it's the name of a company?

Also, `infrastrcure` -> `infrastructure` (typo)